src/python/py/models/builder.py

```diff
@@ -2036,13 +2048,14 @@ def make_model(self, input_path):
                 kv_size,
                 self.intermediate_size,
                 self.num_layers,
+                self.extra_options["adapter_path"] if "adapter_path" in self.extra_options else None,
             )
         else:
             # Load PyTorch model
             extra_kwargs = {"num_hidden_layers": self.num_layers} if "num_hidden_layers" in self.extra_options else {}
             model = AutoModelForCausalLM.from_pretrained(self.model_name_or_path, cache_dir=self.cache_dir, token=self.hf_token, trust_remote_code=True, **extra_kwargs)
 
-        if "adapter_path" in self.extra_options:
+        if "adapter_path" in self.extra_options and self.quant_type is None:
             from peft import PeftModel
```

Comment:
Can we add a comment specifying which case this is (since this is not part of the above `if-elif-else` block)?